assets/README.rst

```diff
@@ -36,7 +36,7 @@ Community
 
 You can talk to the community through:
```

Comment:
I'm not sure what this file is for... I can't think of any reason we'd need a README in an assets directory.

Reply:
Not sure how this file ended up in there. Removing it for this branch.